README.md

```diff
@@ -458,6 +459,24 @@ knpu_oauth2_client:
             # whether to check OAuth2 "state": defaults to true
             # use_state: true
 
+        # will create service: "knpu.oauth2.client.apple"
+        # an instance of: KnpU\OAuth2ClientBundle\Client\Provider\AppleClient
+        # composer require patrickbussmann/oauth2-apple
+        apple:
+            # must be "apple" - it activates that type!
+            type: apple
+            # add and configure client_id and client_secret in parameters.yml
+            client_id: '%env(OAUTH_APPLE_CLIENT_ID)%'
+            client_secret: '%env(OAUTH_APPLE_CLIENT_SECRET)%'
```

Comment:
No `client_secret`? We need to find a way to remove this line then to avoid confusion. I wonder if we have any other clients in this bundle that don't have `client_secret` as well, how do we remove this field from docs?

Reply:
at beginning i removed it and added the needed custom params, but it causes the build to fail ><
i've also tried to add a comment on the line, same build fails..

i dunno :/